shared/test/index.html

```diff
@@ -57,6 +57,7 @@
   <script src="/shared/js/textChatView.js"></script>
   <script src="/shared/js/urlRegExps.js"></script>
   <script src="/shared/js/linkifiedTextView.js"></script>
+  <script src="/shared/js/remoteCursorStore.js"></script>
```

Comment:
You'll need to update karma.*.js files to have it load remoteCursorStore.js for add-on and standalone tests:

https://github.com/mozilla/loop/blob/master/test/karma/karma.coverage.desktop.js#L34
`+    "built/add-on/chrome/content/shared/js/remoteCursorStore.js",`

https://github.com/mozilla/loop/blob/master/test/karma/karma.coverage.shared_standalone.js#L35
`+    "built/standalone/content/shared/js/remoteCursorStore.js",`